README.MD

```diff
@@ -62,4 +62,13 @@ The response for both cases will be like that:
 
 ## Command usage
 
-You can use de command `application-health:check-health` to check health in your cli application
+You can use the follow commands to check healthy in your cli application:
+
+```sh
+php artisan application-health:check-health
```

Comment:
Are we not going to deprecate this command? If so, there's no reason for us to share it with users.

Reply:
I don't think about deprecation. Maybe yes.